tests/lib/nested.sh

```diff
@@ -3,6 +3,10 @@
 # shellcheck source=tests/lib/systemd.sh
 . "$TESTSLIB"/systemd.sh
 
+WORK_DIR=/tmp/work-dir
```

Comment:
How about `mktemp -d` instead? This helps avoid accidental clashes. You can then get rid of `mkdir` below.

Reply:
The problem is that we source this file in many places and we don't want to make mktemp more than once. Any idea about how to deal with this?

Reply:
Ah, I see, I missed that. I guess it has to stay as is then.